docs/source/guides/contributor/chapters/environment.rst

```diff
@@ -3,14 +3,18 @@ Development environment
 
 .. include:: /.include/helpus.rst
 
-.. warning:: TODO: Needs improvment here. i.e: virtualenvs, GPG, etc.
+.. warning:: TODO: Needs improvement here. i.e: virtualenvs, GPG, etc.
 
 Installing dependencies
 -----------------------
 
 You need to install few dependencies before start coding::
 
- $ sudo dnf install gcc libvirt-devel
+ $ sudo dnf install gcc libvirt-devel python-devel python-enchant python-pip pylint
```

Comment:
Ideally this should be a list of system distro dependencies only. `pyenchant` and `pylint` are already on `requirements-selftests.txt`.

Reply:
Personally, I use all packages from the system and in the beginning, I always had a hard time trying to find which package was missing.

Reply:
I agree that is confusing, but having a few packages here and others in `requirements-selftests.txt` IMO make things more confusing.

In an ideal python package, requirements should be installed automatically as much as possible for automatic tasks, deployments and follow some conventions for new contributors.

Maybe the installing dependencies section should  instruct developers to run `make requirements-selftests` or `pip install -r requirements-selftests.txt`, in that way we will also be less prune to future mistakes here in case we forget to update the docs.

Reply:
> Maybe the installing dependencies section should instruct developers to run `make requirements-selftests` or `pip install -r requirements-selftests.txt`, in that way we will also be less prune to future mistakes here in case we forget to update the docs.

This or all the packages listed here would help. Leaving one or two packages here and no other information does not help much.

Reply:
I tend to use directly the packages from the system, but this is for the setup of a development environment so it makes sense to point developers to use `pip` with `requirements-selftests.txt`. 
I'll update the list of packages from the system anyway because `python-devel` is required and add a comment about   `pip` with `requirements-selftests.txt`